snap/special.go

```diff
@@ -0,0 +1,65 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2016 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package snap
+
+import (
+	"fmt"
+)
+
+var implicitSlots = []string{
+	"firewall-control",
+	"home",
+	"locale-control",
+	"log-observe",
+	"mount-observe",
+	"network",
+	"network-bind",
+	"network-control",
+	"network-observe",
+	"snap-control",
+	"system-observe",
+	"timeserver-control",
+	"timezone-control",
+	// XXX: those two should perhaps not be added by default
```

Comment:
I'll add a card for that and do that after 16.04, for now this just gets us going and if we release the os snap again with properly defined slots we can drop this feature altogether.